src/tscore/unit_tests/test_MMH.cc

```diff
@@ -83,15 +83,18 @@ TEST_CASE("MMH", "[libts][MMH]")
       printf("********** collision %d\n", xy);
   }
 
-  unsigned char *s  = (unsigned char *)MMH_x;
-  int l             = sizeof(MMH_x);
-  unsigned char *s1 = (unsigned char *)ats_malloc(l + 3);
+  unsigned char *s       = (unsigned char *)MMH_x;
+  int l                  = sizeof(MMH_x);
+  unsigned char *s1      = (unsigned char *)ats_malloc(l + 4);
```

Comment:
Makes sense. What do you think of using `sizeof(int32_t)` instead of `4`? It would make the intention more explicit.

`unsigned char *s1      = (unsigned char *)ats_malloc(l + sizeof(int32_t));`

Reply:
I made a change to use sizeof(uint32_t)